### PR TITLE
New package: keyd-2.4.0

### PR DESCRIPTION
**Pull request by Barbaross93**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86-64)

### CODE REVIEW

**Comment by Barbaross93** [comment]
Closes #37175

File: srcpkgs/keyd/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'keyd'
+pkgname=keyd
+version=2.4.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+short_desc="Key remapping daemon for linux"
+maintainer="Barbaross <cullenrss@gmail.com>"
+license="MIT"
+homepage="https://github.com/rvaiya/keyd"
```
Comment (classabbyamp) [inline srcpkgs/keyd/template:10]:
```suggestion
homepage="https://github.com/rvaiya/keyd"
changelog="https://raw.githubusercontent.com/rvaiya/keyd/master/docs/CHANGELOG.md"
```

**Comment by Barbaross93** [comment]
@classabbyamp Thanks for the suggestions! I believe everything should be all set.

**Comment by Barbaross93** [comment]
Huh, I'm not sure what the failed check is about:
```
Cloning into bare repository '/github/home/.cache/xlocate.git'...
warning: templates not found in /usr/share/git-core/templates
fatal: unable to access 'https://alpha.de.repo.voidlinux.org/xlocate/xlocate.git/': Failed to connect to alpha.de.repo.voidlinux.org port 443 after 130470 ms: Connection timed out
fatal: cannot change to '/github/home/.cache/xlocate.git': No such file or directory
Error: Process completed with exit code 128.
```

**Comment by classabbyamp** [comment]
That can be ignored for now, there's some downtime in void's infra at the moment.

**Pull request merged**